### DISCUSSION TITLE
KeeWeb email support group

### DISCUSSION THREAD

**Discussion by antelle** [keeweb/keeweb#2171] [General Discussion]
Hi! I made a google group for handling emails sent to keeweb@keeweb.info.
Added @Aetherinox there for now.
@HarlemSquirrel @SethFalco if you would like to be added as well, you can share your email with me.

We can use the group for private conversation between project maintainers.
Also my private contact should be updated in the app's Help section to the group's email so that we all can see the requests sent there.

**Comment by Aetherinox** [comment]
Thanks. I joined / popped in there.

I need to stop Github emails for other products I am starred to. I am getting roughly 50 emails a day, and it's getting hard to keep track of emails. I just cleared out 2000.
